packages/firebase/firestore/memory/index.ts

```diff
@@ -20,4 +20,4 @@
  * `firebase/firestore/memory`.
  */
 
-import '../../../firestore/dist/index.memory.esm';
+import '../../../firestore/dist/index.memory.esm2017';
```

Comment:
hmm, we should import from `@firebase/firestore` instead of from the local path. The build tool will choose the build according to the configuration.

```
import '@firebase/firestore/memory'
```